cpan/Encode/t/Unicode.t

```diff
@@ -25,7 +25,7 @@ use Encode qw(encode decode find_encoding);
 
 #
 # see
-# http://www.unicode.org/unicode/reports/tr19/
+# http://www.unicode.org/reports/tr19/
```

Comment:
This file is also maintained on CPAN, in the same distribution.

Reply:
Same https://github.com/dankogai/p5-encode/pull/158